applications/BurveProtocol.md

```diff
@@ -0,0 +1,243 @@
+
+
+## Project Overview :page_facing_up:
+### Overview
+- Project Name: Burve Protocol
+- Tagline: The AMM 3.0 that innovates bonding curves for fair, LP-less, unruggable, and instantly tradable token launches, flexible and streamlined unilateral liquidity management, and a more efficient DeFi ecosystem.
+
+#### Brief Description
+Burve Protocol is a cutting-edge AMM protocol designed to support a wide range of cryptocurrency use cases, from simple memecoin launches to sophisticated AMM solutions. Our protocol addresses the limitations of existing AMMs, such as initial liquidity challenges and potential manipulation, by integrating bonding curves inspired by S. De la Rouvier’s work. This allows for a more inclusive and efficient financial ecosystem, offering a seamless pathway for a token’s journey from creation and minting to exchange.
+
+#### Integration with Substrate / Polkadot / Kusama
+Burve Protocol leverages Substrate to build a robust, scalable, and interoperable solution within the Polkadot and Kusama ecosystems. By utilizing Substrate, we ensure that our protocol benefits from the security, scalability, and interoperability inherent in the Polkadot network. This integration allows Burve Protocol to connect with other parachains, fostering a more cohesive and efficient DeFi landscape.
+
+#### Team's Motivation
+Our team comprises strong believers in innovation, DeFi, and the power of mathematics. We trust that combining mathematical principles with blockchain technology can lead to greater and more efficient financial freedom. While AMM solutions like Uniswap and order book AMMs have advanced the DeFi revolution, they also have significant drawbacks. Issues such as initial liquidity challenges, liquidity depletion, and potential manipulation by vested entities have highlighted the need for improvement.
+
+Inspired by the potential of bonding curves, as discussed in [S. De la Rouvier's blog post](https://medium.com/@simondlr/a-bonded-curation-community-247f14a6de04), our researchers saw an opportunity to create a more inclusive financial future. We aim to integrate bonding curves into a wider range of applications, providing a holistic solution for token creation, minting, and exchange. This vision led to the creation of Burve Protocol, where we strive to offer innovative solutions to the current limitations of AMMs, thereby fostering a more inclusive and efficient DeFi economy.
+
+Currently, there is no similar solution in the Polkadot ecosystem. As an indispensable infrastructure for the ecosystem, we believe that the services we provide are necessary, efficient, and essential for users. We can genuinely bring more interactions, transactions, smart contract deployments, and more users to Polkadot.
+
+### Project Details
+#### UI
+Our MVP is based on Solidity and is already live, and the Polkadot version will be developed based on the Substrate tech stack and Polkadot structure with the same UI. [Check UI here](https://app.burve.io/).
```

Comment:
With the plans for smart contracts on AssetHub it will be possible to compile your Solidity code to PolkaVM so that you can simply port your project to Polkadot. [See the roadmap here](https://forum.polkadot.network/t/contracts-on-assethub-roadmap/9513). Have you considered waiting for that moment? Given that Parity plans to achieve this until EOY it sounds like a wasted effort to me to now rewrite your whole tech to work on substrate. This way you could also conveniently integrate with all the assets already living on AssetHub so that you suffer no liquidity fragmentation which you raised as a concern yourself earlier in the application. Furthermore given that this transition towards enabling smart contracts on AssetHub will be iterative and starting on the testnet Westend, you could align your project with this strategy and do an experimental deployment on Westend or Kusama so that you can validate your project instead of having to deploy to Polkadot directly. What do you think?